pkg/ddl/util/schema_change_notifier.go

```diff
@@ -27,15 +27,14 @@ import (
 // check the GetType of SchemaChange and call the corresponding getter function
 // to retrieve the needed information.
 type SchemaChangeEvent struct {
-	// todo: field and method will be added in the next few pr on demand
 	tableInfo       *model.TableInfo
 	oldTableInfo    *model.TableInfo
 	addedPartInfo   *model.PartitionInfo
 	droppedPartInfo *model.PartitionInfo
 	columnInfos     []*model.ColumnInfo
-	// nonPartTableID is used to store the non-partitioned table that is converted to
-	// a partitioned table in NewAddPartitioningEvent.
-	nonPartTableID int64
+	// oldTableID4Partition is used to store the table ID when a table transitions from being partitioned to non-partitioned,
+	// OR vice versa.
```

Comment:
```suggestion
	// or vice versa.
```

It is a comma, so I guess it would be lowercase.